Assets/Plugins/iOS/Heyzap/HeyzapAds.h

```diff
@@ -325,6 +325,22 @@ extern NSString * const HZFyberVirtualCurrencyResponseUserInfoKey;
  */
 + (void) setGDPRConsent:(BOOL)consentGiven;
 
+/**
+ *  Sets the consent data.
+ *
+ *  @param gdprConsentData The consent data
+ *
+ *  @since v9.21.0
```

Comment:
Shouldn't it be `since 9.22.0`? I mean, the next version that will be released, no?